components/CardForm/index.js

```diff
@@ -141,25 +182,64 @@ export default function CardForm({
             required
           />
           <StyledLabel htmlFor="date">Date: </StyledLabel>
-          <StyledInputField
-            type="date"
-            id="date"
-            name="date"
-            autoComplete="off"
-            min={getCurrentDate()}
-            defaultValue={existingActivityData?.date || ""}
-            required
-          />
+          <StyledDateWrapper>
+            <StyledInputField
+              type="text"
+              id="date"
+              name="date"
+              autoComplete="off"
+              value={
+                existingActivityData?.date
+                  ? existingActivityData.date
+                  : selectedDate
+                  ? format(selectedDate, "yyyy-MM-dd")
+                  : getCurrentDate()
```

Comment:
If you implemented the suggestion above then you could do the following. 

```suggestion
               date
                  ? date
                  : selectedDate
                  ? format(selectedDate, "yyyy-MM-dd")
                  : getCurrentDate()
```